code/modules/atmospherics/machinery/components/unary_devices/machine_connector.dm

```diff
@@ -0,0 +1,110 @@
+///To be used when there is the need of an atmos connection without repathing everything (eg: cryo.dm)
+/obj/machinery/atmospherics/components/unary/machine_connector
```

Comment:
Given this is a separate object on the same loc, does that mean you can interact with it on its own? Attack it to destroy it? Unwrench it?

Would it be smart to make it indestructible / `Obscured` to prevent people from messing with it?

Reply:
You can't unwrench it, but you can interact with it, how does the Obscured work?

Reply:
See `IsObscured`, essentially it's what prevents you from using alt click turf view from clicking on grilles when there's a window on top. You could override it for your connector to prevent people from clicking on it or messing with it.